### ISSUE TITLE
Can't create space

### ISSUE THREAD

**Issue by ahmedwaleedmalik** [kiosk-sh/kiosk#62]
Hi, I am testing out kiosk on openshift and trying to create a space as shown in the tutorial at

https://github.com/kiosk-sh/kiosk#32-create-spaces

But I am getting the following error:

```
Error from server (Conflict): error when creating "https://raw.githubusercontent.com/kiosk-sh/kiosk/master/examples/space.yaml": Operation cannot be fulfilled on namespaces "johns-space": the object has been modified; please apply your changes to the latest version and try again
```

**Comment by FabianKramm** [comment]
@ahmedwaleedmalik thanks for reporting! I'll fix this right away!

**Comment by FabianKramm** [comment]
@ahmedwaleedmalik should be fixed with v0.1.10
